<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="0" total="6" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-08-16T19:08:57 IST" name="Suite" finished-at="2023-08-16T19:10:13 IST" duration-ms="76383">
    <groups>
      <group name="ErrorValidations">
        <method signature="ErrorValidationsTest.ProductValidation()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" name="ProductValidation" class="Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest"/>
        <method signature="ErrorValidationsTest.loginErrorValidation()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" name="loginErrorValidation" class="Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest"/>
      </group> <!-- ErrorValidations -->
      <group name="Purchase">
        <method signature="SubmitOrderTest.submitOrder(java.util.HashMap)[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" name="submitOrder" class="Automation.SeleniumrepeatProject.Tests.SubmitOrderTest"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2023-08-16T19:08:57 IST" name="Submit Order Test" finished-at="2023-08-16T19:09:44 IST" duration-ms="47319">
      <class name="Automation.SeleniumrepeatProject.Tests.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:08:57 IST" name="launchApplication" finished-at="2023-08-16T19:09:09 IST" duration-ms="11141" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:09:09 IST" name="submitOrder" data-provider="getData" finished-at="2023-08-16T19:09:25 IST" duration-ms="16120" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Mkyasv@1234, ProductName=ZARA COAT 3, CountryName=india, username=shreyasdoshi48@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:09:25 IST" name="TearDown" finished-at="2023-08-16T19:09:25 IST" duration-ms="245" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:09:25 IST" name="launchApplication" finished-at="2023-08-16T19:09:32 IST" duration-ms="7060" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:09:32 IST" name="submitOrder" data-provider="getData" finished-at="2023-08-16T19:09:36 IST" duration-ms="4106" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Ssd@12345, ProductName=ADIDAS ORIGINAL, CountryName=aus, username=sswarali05@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:09:36 IST" name="TearDown" finished-at="2023-08-16T19:09:36 IST" duration-ms="219" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:09:36 IST" name="launchApplication" finished-at="2023-08-16T19:09:43 IST" duration-ms="6289" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OderDisplay()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:09:43 IST" name="OderDisplay" finished-at="2023-08-16T19:09:44 IST" depends-on-methods="Automation.SeleniumrepeatProject.Tests.SubmitOrderTest.submitOrder" duration-ms="1322" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OderDisplay -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.SubmitOrderTest@67d18ed7]" started-at="2023-08-16T19:09:44 IST" name="TearDown" finished-at="2023-08-16T19:09:44 IST" duration-ms="259" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- Automation.SeleniumrepeatProject.Tests.SubmitOrderTest -->
    </test> <!-- Submit Order Test -->
    <test started-at="2023-08-16T19:09:45 IST" name="Error Valications Test" finished-at="2023-08-16T19:10:13 IST" duration-ms="28272">
      <class name="Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:09:45 IST" name="launchApplication" finished-at="2023-08-16T19:09:52 IST" duration-ms="6786" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductValidation()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:09:52 IST" name="ProductValidation" finished-at="2023-08-16T19:09:55 IST" duration-ms="2750" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductValidation -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:09:55 IST" name="TearDown" finished-at="2023-08-16T19:09:55 IST" duration-ms="232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:09:55 IST" name="launchApplication" finished-at="2023-08-16T19:10:01 IST" duration-ms="6100" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="loginErrorValidation()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:10:01 IST" name="loginErrorValidation" finished-at="2023-08-16T19:10:05 IST" duration-ms="3647" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorret email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorret email or password.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest.loginErrorValidation(ErrorValidationsTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:10:05 IST" name="TearDown" finished-at="2023-08-16T19:10:05 IST" duration-ms="268" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:10:05 IST" name="launchApplication" finished-at="2023-08-16T19:10:12 IST" duration-ms="6742" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginErrorValidation()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:10:12 IST" name="loginErrorValidation" finished-at="2023-08-16T19:10:12 IST" duration-ms="730" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorret email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorret email or password.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest.loginErrorValidation(ErrorValidationsTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginErrorValidation -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest@31c7528f]" started-at="2023-08-16T19:10:13 IST" name="TearDown" finished-at="2023-08-16T19:10:13 IST" duration-ms="358" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- Automation.SeleniumrepeatProject.Tests.ErrorValidationsTest -->
    </test> <!-- Error Valications Test -->
  </suite> <!-- Suite -->
</testng-results>
